app/src/main/java/us/shandian/giga/ui/fragment/MissionsFragment.java

```diff
@@ -208,6 +208,14 @@ public boolean onOptionsItemSelected(MenuItem item) {
         }
     }
 
+    public void areYouSureToDeleteAllFiles(){
+        AlertDialog.Builder prompt = new AlertDialog.Builder(mContext);
+        prompt.setTitle("Are you sure you want to delete all your files ?");
```

Comment:
Space before `?` needs to be removed

Reply:
Also, it must be moved into a string ressource to be translatable.